src/unix/fs.c

```diff
@@ -143,6 +143,18 @@ extern char *mkdtemp(char *template); /* See issue #740 on AIX < 7 */
   while (0)
 
 
+static int uv__fs_close(int fd) {
+  int rc;
+
+  rc = close(fd);
+  if (rc == -1)
+    if (errno == EINTR || errno == EINPROGRESS)
+      rc = 0;  /* The close is in progress, not an error. */
+
+  return rc;
```

Comment:
Use `UV__ERR()` in case of error?

Reply:
This is called by `uv__fs_work()` which looks at the return code and stores `UV__ERR(errno)` when it's -1.

Reply:
:+1:  Right.